efficientdet/keras/efficientdet_arch_keras.py

```diff
@@ -351,8 +325,6 @@ def get_config(self):
         'target_width': self.target_width,
```

Comment:
We could remove ```get_config``` if we don't use keras model.

Reply:
I will keep this for now since this PR is already large.